packages/test/stochastic-test-utils/src/describeFuzz.ts

```diff
@@ -24,12 +24,12 @@ function createSuite<TArgs extends StressSuiteArguments>(
  */
 export interface StressSuiteArguments {
 	/**
-	 * Whether the current run is a stress run, which generally runs longer or more programatically generated tests.
-	 *
-	 * Packages which use this should generally have a `test:stress` script in their package.json for conveniently running
-	 * the equivalent checks locally.
+	 * It indicates the "stress level" of tests. A value of 0 means the test is not run in stress mode, while
+	 * higher values indicate increasingly stressful testing. This value also acts as a multiplier for the
```

Comment:
Rather than make this numeric, I'd prefer we use an enum or string union with well-defined semantics for its members. Ideally a test author should be able to look at the docs here and make informed decisions on how to write their tests. The semantics should be in terms of the expectations for

- roughly how long should the tests take?
- are the tests expected to be evergreen? (true for deterministic seeds, false for random seeds)

in addition to other criteria you might find relevant. Because this doc is FF internal, I think it's fine if the documentation for those modes mentions the strategy we use for running tests in that mode (e.g. if we go with the strategies we discussed, we document here that a 'short' mode is run on all CI runs as part of the package's normal correctness tests, a 'medium' is run post check-in when a package is changed, and a 'long' is run on a fixed schedule using random seeds)

Reply:
all the other changes in this PR look fine at a glance, I can review in more detail for places where there are missing changes of a similar sort (if any) and/or nitpick. but it generally looks like what I expected.

Reply:
I have updated the PR and included the unit tests for introducing enum stress modes. For some tests, like `optionalChangeRebaser.test.ts`, parameters such as `numberOfEditsToRebaseOver` should be customized by the area expert or owner, depending on the stress mode. Currently, I prefer to leave these parameters as they are, so they will only differ when stress mode is enabled as before.

Reply:
this doc comment needs to be updated, and the semantics of each value of `StressMode` should be as well.

Reply:
I think the `undefined` stressMode is still necessary, indicating that the case where previous `isStress = false`, i.e. no stress tests to be run

Reply:
it's not obvious whether `Normal` means "no stress" or "normal amount of stress". If it means the former, then I don't see how that's different from `undefined`. If it means something more like the latter, I'd want the semantics of that mode more clearly documented.

Reply:
Yes the document of stressMode is not that correct, have updated it